doc/walkthrough.rst

```diff
@@ -0,0 +1,109 @@
+Walkthrough
+===========
+
+This document walkthroughs a simple scenario with Commissaire.
+
+Configuring a ContainerManager (Optional)
+-----------------------------------------
+If you will be using OpenShift, OCP, or Kubernetes then configuring a ContainerManager is the first thing to do.
+This essentially will tell commissaire how to communicate with your ContainerManager. When a cluster is associated
+to this ContainerManager new hosts will be automatically added into the the ContainerManager as nodes.
+
+Let's say you wanted to add a ContainerManager called ``ocp``,  which has a url of ``https://openshift.example.com``,
+and uses a token of ``aaa`` for authentication:
+
+.. include:: examples/commctl_create_container_manager.rst
+
```

Comment:
OK. My commissaire service has a ContainerManager called `ocp` which is connected to my openshift instance.

Inferring from language used in the paragraph in the previous paragraphs, is the following correct? I am not sure if I understand this fully.

* "new hosts will be automatically added into the ContainerManager as nodes"
* I scale up my OCP cluster. There is now a new node running pods.
* Now my ContainerManager has another host in it?

Or does that language, "added into ... as nodes" imply that upon **first association** with OCP that all hosts in the connected OCP cluster will be added **once** as hosts in commissaire, configured as `openshift nodes`?